Deploy-with-tls.md

```diff
@@ -0,0 +1,84 @@
+    ![Generate password](./media/brypt-password-generation.png)
+1. Head to the tls-manifest/manifests/common/dex/base/config-map.yaml file and update the hash value there (around line 22) with the hash you just generated. You can also change the email address, username and userid. In addition, you can setup multiple users by adding more users to the array.
+1. Update your auth.md file with the new email address and password (plain text password not hash) or store the secrets in a more secure way
+1. Copy the contents of this newly updated manifests folder to the kubeflow manifests folder. This will update the files so the deployment includes your config changes.
+    ```bash
+    cp -r tls-manifest/manifests/ manifests/
+    ```
+1. cd to the manifests folder and install kubeflow
+    ```bash
+    cd manifests
+    ```
+    Install all of the components via a single command
+    
+    ```bash
+    while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+    ```  
+1. Once the command has completed, check the pods are ready
+    
+    ```bash
+    kubectl get pods -n cert-manager
+    kubectl get pods -n istio-system
+    kubectl get pods -n auth
+    kubectl get pods -n knative-eventing
+    kubectl get pods -n knative-serving
+    kubectl get pods -n kubeflow
+    kubectl get pods -n kubeflow-user-example-com
+    ```
+1. Restart dex to ensure everything is working as expected
+    ```bash
+    kubectl rollout restart deployment dex -n auth
+    ```
+1. Configure TLS. Start by getting IP address of istio gateway
+    ```bash
+    IP=$(kubectl -n istio-system get service istio-ingressgateway --output jsonpath={.status.loadBalancer.ingress[0].ip})
+    ```
+    Replace the IP address in the tls-manifest\certificate.yaml file with the IP address of the istio gateway using the sed command below 
+    > :warning: If you are using a mac you will need to change the command to `sed -i '' "s/192.168.0.5/$IP/" tls-manifest/certificate.yaml `. 
+    
+    > :bulb: If these sed commands don't work for any reason or if you don't have sed installed, you will need to update these files manually by replacing the placeholders in the files mentioned below.
+    ```bash
+    cd ..
+    sed -i  "s/192.168.0.5/$IP/" tls-manifest/certificate.yaml 
+    ```
+1. Please note that instead of providing the IP address like we did above, you could give the LB an Azure sub-domain and use that too. Deploy the certificate manifest file.
```

Comment:
```suggestion
1. Please note that instead of providing the IP address like we did above, you could give the LoadBalancer an Azure sub-domain (via the annotation in tls-manifest/manifests/common/istio-1-14/istio-install/base/patches/service.yaml ) and use that too. Deploy the certificate manifest file.
```